src/app/pages/vendor-dashboard/components/vendor-filter/vendor-filter.component.html

```diff
@@ -1 +1,27 @@
-<p>vendor-filter works!</p>
+
+<p style="font-size: 30px;" id="SIc"> Status</p>
+<mat-checkbox color="primary" class="CheckBox"></mat-checkbox>
+<p style="font-size: 15px;">status 1</p>
+<mat-checkbox color="primary" class="CheckBox"></mat-checkbox>
+<p style="font-size: 15px;">status 2</p>
+<mat-checkbox color="primary" class="CheckBox"></mat-checkbox>
+<p style="font-size: 15px;">status 3</p>
+
+<hr>
+<p style="font-size: 30px;">Work Types
```

Comment:
I think the p tag should be itself then followed by the form field rather then wrapping it